asciidoctorj-api/src/main/java/org/asciidoctor/Options.java

```diff
@@ -7,11 +7,15 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * AsciidoctorJ conversion options. Each one maps to an option in Asciidoctor.
+ * See https://docs.asciidoctor.org/asciidoctor/latest/api/options/ for further
+ * details.
+ */
 public class Options {
 
     public static final String IN_PLACE = "in_place";
     public static final String ATTRIBUTES = "attributes";
-    public static final String HEADER_FOOTER = "header_footer";
```

Comment:
I wonder if we want to at least keep the constant?
Even if the option is not recommended, Asciidoctor itself still handles it.

Reply:
I thought about it, my rationale was that if this is not the recommended way, we should not make it easy to use. Users can still use it as a String if they want, but since we had no references in the current code we could remove it.

We could add it back, I admit I am being very strict with the "let's clean thing now that we can to breaking changes" :smile: